### PR TITLE
Feature1 Setup Project

### PR DESCRIPTION
**Pull request by RolandM99**
- Use Gitflow
- Create an npm project with Webpack
- Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
-# setUp-webpack-boilerplate
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service
+
+![screenshot](./screenshot.PNG)
```
Comment (Mihndim2020) [inline README.md]: - [ ] The link to the screenshot is broken, please kindly fix, as this will make your readme file look more professional.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Gitflow followed.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below and under the review._

- [ ] I am unable to run the project from my end, please make sure the project is running and resubmit for review. If possible add a live demo link. See screenshot below

roland

- [ ] It is a good thing you added `node_modules` to `.gitignore` file, please kindly delete it from your repository as it makes your repository dirty. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (RolandM99) [reply README.md]: Hello, dear reviewer!
As you told me in the previous review,  I fixed that error! Kindly try again the `npm install` then `npm start` to run the application or just follow the link I add in my readme file.

**Comment by Davidosky007** [comment]
## Hi @RolandM99 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
good-job-gif



## Cheers and Happy coding!👏👏👏

## Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
## OPTIONAL 
## * To remove the `node modules` folder completely from your repo kindly run these commands ` git rm -r --cached node_modules`

## `git commit -am "node_modules be gone!"`

## `git push origin your_branch_name`

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**